### PR TITLE
NO-ISSUE: fixes git-sha and lib version display values

### PR DESCRIPTION
**Pull request by jkilzi**
This PR fixes the values displayed (in dev-mode) by the env. vars REACT_APP_VERSION and REACT_APP_GIT_SHA.  

- Removes old scripts that are no longer in use from the hacks folder
- REACT_APP_VERSION now displays the version of the ui-lib located inside the node_modules
- REACT_APP_GIT_SHA show the assisted-ui's current git-sha
- REACT_APP_API_URL needs to be defined inside a .env.development.local file (not kept under VCS, so every dev can have their own settings)

### CODE REVIEW

**Review by ammont82 — APPROVED** [review]

File: hacks/setup-env.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+REACT_APP_API_URL="http://REACT_APP_API_URL.env.variable.not:set"
```
Comment (celdrake) [inline hacks/setup-env.sh:1]: Shouldn't the script include the `#!/bin/bash` directive?

File: hacks/setup-env.sh [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+REACT_APP_VERSION="$(node -e 'console.log(require("./node_modules/openshift-assisted-ui-lib/package.json").version)')"
```
Comment (celdrake) [inline hacks/setup-env.sh:2]: Maybe this a faster alternative?

`cat node_modules/openshift-assisted-ui-lib/package.json | jq -r '.version'`

**Comment by celdrake** [comment]
@jkilzi We should update the file `create-test.mjs` as it is being referenced in the initial comment with its previous name `setup-env.mjs`

File: hacks/setup-env.sh (hunk 1 above)
Comment (jkilzi) [reply hacks/setup-env.sh:1]: no because this file is meant to be sourced, no shell will execute the code

File: hacks/setup-env.sh (hunk 2 above)
Comment (jkilzi) [reply hacks/setup-env.sh:2]: Yes, but that assumes that you are using Linux and have `jq` installed.
No matter what OS you will use in order to execute the project, you must have node installed there.

File: hacks/setup-env.sh (hunk 1 above)
Comment (celdrake) [reply hacks/setup-env.sh:1]: That's true, thanks

File: hacks/setup-env.sh (hunk 2 above)
Comment (celdrake) [reply hacks/setup-env.sh:2]: True also, specially `jq` would not be available in most cases.

**Review by celdrake — APPROVED** [review]

**Pull request merged**